Partition (WBP)/Lists/I0049B.lsts.yml

```diff
@@ -0,0 +1,15 @@
+﻿List:
```

Comment:
This list is not required. Add all objects to list I0049.
And that list has STILL not been corrected